src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs

```diff
@@ -62,7 +62,7 @@ private static void Initialize()
             }
 
             Logging.LogAdapter = new FileLogAdapter(logPath);
-            Logging.Info("Initializing GitHub for Unity version " + ApplicationInfo.Version);
+            Logging.Info("Initializing GitHubForUnity:v{0} Unity:v{1}", ApplicationInfo.Version, Environment.UnityVersion);
```

Comment:
If you want to have a greppable format for this, I suggest adding single quotes around the versions.